addon/mixins/model-validator.js

```diff
@@ -289,5 +294,11 @@ export default Ember.Mixin.create({
               .replace(/([A-Z])/g, ' $1')
               // uppercase the first character
               .replace(/^./, function(str){ return Ember.String.capitalize(str); });
+  },
+  _isFunction: function(func) {
+    return typeof func === 'function';
```

Comment:
@dmuneras Let's use [Ember methods](http://emberjs.com/api/#method_typeOf) here. Same for the function below.